product-service/sql/DDL.sql

```diff
@@ -0,0 +1,16 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE products
+(
+    id          UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    title       TEXT NOT NULL,
+    description TEXT,
+    price       INTEGER
+);
```

Comment:
It is better to use real type in price

Reply:
I prefer INTEGER, hope it solves floating point problem